oap-server/server-query-plugin/query-graphql-plugin/src/main/java/org/apache/skywalking/oap/query/graphql/resolver/MetricQuery.java

```diff
@@ -40,6 +44,7 @@
  * @since 8.0.0 This query is replaced by {@link MetricsQuery}
  */
 @Deprecated
+@Slf4j
 public class MetricQuery implements GraphQLQueryResolver {
```

Comment:
Please notice, this class has been `Deprecated`, which means it is a shell only. The real codes are in `org.apache.skywalking.oap.query.graphql.resolver.MetricsQuery`